### ISSUE TITLE
Warning for plugin 'cordova-plugin-compat' not being compatible with cordova-android 9

### ISSUE THREAD

**Issue by ghadardwan** [rodrigograca31/Canvas2ImagePlugin#20]
I am getting this warning for adding this plugin 'cordova-plugin-compat' as a dependency for 'canvas2Image'
![image](https://user-images.githubusercontent.com/28740840/133069345-acf4d29e-2046-4232-82fe-d500809b990f.png)

However, due to the documentation related to the compat plugin,
it is deprecated and no longer being worked on as the functionality provided by this plugin is now included in cordova-android 6.3.0
https://www.npmjs.com/package/cordova-plugin-compat
kindly consider removing this dependency, as I guess it's no longer needed, and including it in the applications may cause a problem or unexpected behavior in the future.

**Comment by rodrigograca31** [comment]
Yeah seems its automatically skipped because we use version `^1.0.0` which matches 1.2.0 which auto detects if the plugin is needed or not.

I my self left a comment on the code in the past but didnt actually remove it...
https://github.com/rodrigograca31/Canvas2ImagePlugin/blob/52ec655ad453043f618a68e339927c9579f3d067/plugin.xml#L22
Feel free to send a Pull Request and I will accept it and merge it!
